cargo-process.el

```diff
@@ -126,6 +130,42 @@
       (rename-buffer buffer))
     (set-process-sentinel (get-buffer-process buffer) 'cargo-process--finished-sentinel)))
 
+
+;; I/O tools
+
+
+(defun cargo-process--get-current-test ()
+  "Return the current test."
+  (let ((start (point))
+        name)
+    (save-excursion
+      (end-of-line)
+      (unless (and
+               (search-backward-regexp
+                (s-concat "^[[:space:]]*fn[[:space:]]*") nil t)
+               (save-excursion (go-end-of-defun) (< start (point))))
+        (error "Unable to find test"))
+      (save-excursion
+        (search-forward "fn ")
```

Comment:
Why not search-forward-regexp like the previous one? If you're going to use that regex again you should create a defconst to hold that value (instead of repeating it)